tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -74,22 +126,22 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 	})
 
 	Context("When using from-file option", func() {
+
+		JustBeforeEach(func() {
+			preSetup()
+		})
+
+		JustAfterEach(func() {
+			cleanPreSetup()
+		})
+
 		It("should be able to create a service", func() {
 			// First let's grab the etcd operator's name from "odo catalog list services" output
 			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
-			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]`).FindString(operators)
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(operators)
 
 			stdOut := helper.CmdShouldPass("odo", "service", "create", etcdOperator, "--crd", "EtcdCluster", "--dry-run")
 
-			// change the metadata.name from example to example2 so that we can run tests parallely
-			lines := strings.Split(stdOut, "\n")
-			for i, line := range lines {
-				if strings.Contains(line, "name: example") {
-					lines[i] = strings.Replace(lines[i], "example", "example2", 1)
-				}
-			}
-			stdOut = strings.Join(lines, "\n")
-
 			// stdOut contains the yaml specification. Store it to a file
 			randomFileName := helper.RandString(6) + ".yaml"
 			fileName := filepath.Join("/tmp", randomFileName)
```

Comment:
This won't work on windows. Please use context instead.

Reply:
This is not a part of the PR.

Reply:
Yes, I understood but It can be fixed right? It's not a big change that we can go for a new pr and review process. Anyway in the commit message you can mention the fix so that we can find out what this pr fixes.

Reply:
Yes, I understood but It can be fixed right? It's not a big change that we can go for a new pr and review process. Anyway in the commit message you can mention the fix so that we can find out what this pr fixes.

Reply:
I have not checked if it's a big change or a small one. Last time I tried to fix something in my PR that had not been touched in my PR, it ended up going for a really long time. So, if you think that this line of code is problematic, I humbly request you to open a new issue. But I'm not addressing it in this PR.

Reply:
Issue has been tracked in https://github.com/openshift/odo/issues/3246